packages/react-components/react-tree/src/components/TreeItemLayout/renderTreeItemLayout.tsx

```diff
@@ -4,19 +4,23 @@
 import { createElement } from '@fluentui/react-jsx-runtime';
 import { getSlotsNext } from '@fluentui/react-utilities';
 import type { TreeItemLayoutState, TreeItemLayoutSlots } from './TreeItemLayout.types';
+import { ButtonContextProvider } from '@fluentui/react-button';
 
 /**
  * Render the final JSX of TreeItemLayout
  */
 export const renderTreeItemLayout_unstable = (state: TreeItemLayoutState) => {
-  const { isActionsVisible } = state;
   const { slots, slotProps } = getSlotsNext<TreeItemLayoutSlots>(state);
   return (
     <slots.root {...slotProps.root}>
+      {slots.expandIcon && <slots.expandIcon {...slotProps.expandIcon} />}
       {slots.iconBefore && <slots.iconBefore {...slotProps.iconBefore} />}
       {slotProps.root.children}
       {slots.iconAfter && <slots.iconAfter {...slotProps.iconAfter} />}
-      {!isActionsVisible && slots.aside && <slots.aside {...slotProps.aside} />}
+      <ButtonContextProvider value={state.buttonContextValue}>
+        {state.isAsideVisible && slots.aside && <slots.aside {...slotProps.aside} />}
```

Comment:
Also when the mouse leaves the menu, the entire actions is no longer rendered which results in the menu being dismissed. It's not a big deal for me but just wanted to point it out, because this is not what the Teams chat list does
![actions menu dismiss](https://github.com/microsoft/fluentui/assets/20744592/d8333652-dd79-4271-841e-bb0790e3d15c)

Reply:
This one I'm aware. we'll need to add a listener to the document to ensure this behaviour properly, I'm avoiding this for now, but I'll look into it